music_assistant/server/providers/radiobrowser/__init__.py

```diff
@@ -0,0 +1,151 @@
+"""RadioBrowser musicprovider support for MusicAssistant."""
+from __future__ import annotations
+
+from collections.abc import AsyncGenerator
+from time import time
+from typing import TYPE_CHECKING
+
+from radios import RadioBrowser, RadioBrowserError
+
+from music_assistant.common.models.config_entries import ConfigEntry, ConfigValueType
+from music_assistant.common.models.enums import LinkType, ProviderFeature
+from music_assistant.common.models.media_items import (
+    ContentType,
+    ImageType,
+    MediaItemImage,
+    MediaItemLink,
+    MediaType,
+    ProviderMapping,
+    Radio,
+    SearchResults,
+    StreamDetails,
+)
+from music_assistant.constants import __version__
```

Comment:
```suggestion
from music_assistant.constants import __version__ as MASS_VERSION
```

Reply:
meh annoying haha... just put "noqa: N812" behind it because the uppercase constant is nice for readability further in the code